app/Services/GitHub/IssueCreateService.php

```
@@ -0,0 +1,482 @@
+    }
+
+    /**
+     * Get available labels for repository
+     */
+    public function getAvailableLabels(string $repo): array
+    {
+        $this->initializeCache($repo);
+
+        $cacheKey = 'labels';
+        if (! isset($this->memoizedData[$cacheKey])) {
+            [$owner, $repoName] = explode('/', $repo);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$owner'. (Unused Code Rules)

(UnusedLocalVariable)